testing/execution.ts

```diff
@@ -443,22 +460,10 @@ export async function executeFormulaOrSyncWithVM<T extends PackFormulaResult | G
 
   const ivmContext = await ivmHelper.setupIvmContext(bundlePath, executionContext);
 
-  return executeThunk(ivmContext, {params, formulaSpec: formulaSpecification}, bundlePath, bundlePath + '.map') as T;
-}
-
-export class VMError {
```

Comment:
This appears to be dead code, so I removed it. I couldn't find any instance of it being used in this repo or the packs repo.